Tests/Repository/ContainerRepositoryFactoryTest.php

```diff
@@ -63,7 +63,6 @@ public function testCustomRepositoryIsReturned()
 
     /**
      * @expectedException \RuntimeException
-     * @expectedExceptionMessage The service "my_repo" must extend EntityRepository (or a base class, like ServiceEntityRepository).
```

Comment:
This is wrong: using `stdClass` for a repository should still throw an exception since it does not implement `ObjectRepository`.